src/utils/index.ts

```diff
@@ -0,0 +1,54 @@
+import { IUserAccount } from '../contexts/root/root.context';
+import { LOCALHOST_APP_ID, VERCEL_DEPLOYMENT_APP_ID } from './constants';
+
+/**
+ *
+ * @returns {boolean} return true if the window hostname contains `localhost`
+ */
+export const getIsLocalhost = () => {
```

Comment:
Perhaps just making it `isLocalhost` is easier to understand.

Reply:
let's have it this way Please. :pray: